_posts/2019-03-22-team-time.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Team Time: developing efficient team communication"
+author: richard_harper
+---
+
+![Team Time!]({{ "/images/direction.png" }})
+
+# Avoiding the overrunning daily Stand-Up
+
+At Infinity Works we use an agile way of working which usually includes a daily Stand-Up. This 5-10 minute timeboxed meeting allows the team to provide an update and more importantly raise any issues they are facing. Our team uses the format of what they did yesterday (to achieve the team’s goal), what they are working on that day (to achieve the team’s goal), and anything that might prevent them from progressing with their work.
+
+Like most teams, it wasn’t uncommon for us to get side tracked or to start solutioning during a Stand-Up leading to long Stand-Up times and using up team member’s time who might have better things to be doing.
+
+Also finding time where the whole team is free or not buried in their work to discuss something could be tricky. This caused discussions to take place without all of the team and encouraged silo working. Or even worse, discouraged team members from discussing something and becoming the single point of failure.
+
+# Arriving at the solution
+
+We decided to block out some time, 30-60 minutes daily, after a Stand-Up for the team to discuss ideas/solutions, draft weeknotes, or new technical approaches they had found to improve our work. We fondly named this Team Time.
```

Comment:
"or new technical approaches they had found"
During this whole sentence you use we/our but suddenly change to "they", which reads weird.

Reply:
Should be resolved please re-review